packages/api/src/constants/onfido.ts

```diff
@@ -0,0 +1,350 @@
+    label: 'Expiry date',
+    type: 'date',
+    required: true,
+};
+
+export const ONFIDO_PHRASES = {
+    country_select: {
+        alert_dropdown: {
+            country_not_found: 'Country not found',
+        },
+        alert: {
+            another_doc:
+                'Documents from that country are not currently supported — <fallback>try another document type</fallback>',
+        },
+        button_primary: 'Submit document',
+        search: {
+            accessibility: 'Select country',
+            input_placeholder: 'e.g. United States',
+            label: 'Search for country',
+        },
+        title: 'Select issuing country',
+    },
+    cross_device_checklist: {
+        button_primary: 'Submit verification',
+        info: 'Tips',
+        list_item_doc_multiple: 'Documents uploaded',
+        list_item_doc_one: 'Document uploaded',
+        list_item_selfie: 'Selfie uploaded',
+        subtitle: "We're now ready to verify your identity",
+        title: "Great, that's everything we need",
+    },
+    cross_device_error_desktop: {
+        subtitle: 'The link only works on mobile devices',
+        title: "Something's gone wrong",
+    },
+    cross_device_error_restart: {
+        subtitle: "You'll need to restart your verification on your computer",
+        title: "Something's gone wrong",
+    },
+    cross_device_intro: {
+        button_primary: 'Get secure link',
+        list_accessibility: 'Steps required to continue verification on your mobile',
+        list_item_finish: 'Check back here to finish the submission',
+        list_item_open_link: 'Open the link and complete the tasks',
+        list_item_send_phone: 'Send a secure link to your phone',
+        subtitle: "Here's how to do it:",
+        title: 'Continue on your phone',
+    },
+    cross_device_return: {
+        body: 'Your computer may take a few seconds to update',
+        subtitle: 'You can now return to your computer to continue',
+        title: 'Uploads successful',
+    },
+    doc_confirmation: {
+        alert: {
+            blur_detail: 'Make sure everything is clear',
+            blur_title: 'Blurry photo detected',
+            crop_detail: 'Make sure full document is visible',
+            crop_title: 'Cut-off image detected',
+            glare_detail: 'Move away from direct light',
+            glare_title: 'Glare detected',
+            no_doc_detail: 'Make sure all of the document is in the photo',
+            no_doc_title: 'No document detected',
+        },
+        body_id: 'Make sure your card details are clear to read, with no blur or glare',
+        body_image_medium: 'It’ll take longer to verify you if we can’t read it',
+        body_image_poor: 'To smoothly verify you, we need a better photo',
+        body_license: 'Make sure your license details are clear to read, with no blur or glare',
+        body_passport: 'Make sure your passport details are clear to read, with no blur or glare',
```

Comment:
These are not wrapped with `<Localize>` because we would have to install `@deriv/translations` to get the Localize component